code/index.html

```diff
@@ -5,18 +5,111 @@
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <img class="logo" src="Katt-logga.png" alt="logga med kattmotiv">
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&display=swap" rel="stylesheet">
+      <nav class="nav-bar">
+        <a href="https://www.youtube.com/watch?v=k0QhnMJnJE0">Menu</a> 
+        <a href="https://en.wikipedia.org/wiki/Cat">About</a>
+       <a href="https://www.meowparlour.com">Contact</a>
+      </nav> 
     </header>
 
+    <div class="tidningens-logga">
```

Comment:
I don't know if this is something to think about, but maybe stick to one language, so when naming classes maybe stick to english so we get used to using english instead of swedish.